runtime/westend/src/governance/fellowship.rs

```diff
@@ -0,0 +1,355 @@
+// Copyright (C) Parity Technologies (UK) Ltd.
+// This file is part of Polkadot.
+
+// Polkadot is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Polkadot is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Polkadot. If not, see <http://www.gnu.org/licenses/>.
+
+//! Elements of governance concerning the Westend Fellowship.
```

Comment:
for the Westend we should take Polkadot as a reference, where the Fellowship located on Collecitves chain, which is also there for Westend.
for Westend's Collecitves you wont need to do anything, it uses just exactly the same runtime.

Reply:
changing Westend config to follow Polkadot at [cc5cccc](https://github.com/paritytech/polkadot/pull/7272/commits/cc5cccc5521c70d1ce60396a86d09a47fe9be538)